sound/soc/sof/intel/hda.h

```diff
@@ -339,6 +339,13 @@
 #define HDA_ADSP_FW_STATUS_SKL		HDA_ADSP_SRAM0_BASE_SKL
 #define HDA_ADSP_ERROR_CODE_SKL		(HDA_ADSP_FW_STATUS_SKL + 0x4)
 
+/* Host Device Memory Space */
```

Comment:
fix your commit message, typos and no need to describe things you don't use.

ASoC: SOF: Intel: hda: add the SSP Host Device memory space

The DSP SSP device memory can be conditionally accessed by the
host (depending on access policy)

Add the SSP base memory offset of APL and CNL.

Reply:
OK, thanks!